proto/umee/leverage/v1/query.proto

```diff
@@ -356,3 +361,30 @@ message QueryLiquidationTargetsRequest {}
 message QueryLiquidationTargetsResponse {
   repeated string targets = 1;
 }
+
+// QueryMarketSummaryRequest defines the request structure for the
+// MarketSummary gRPC service handler.
+message QueryMarketSummaryRequest {
+  string denom = 1;
+}
+
+// QueryMarketSummaryResponse defines the response structure for the
+// MarketSummary gRPC service handler.
+message QueryMarketSummaryResponse {
+  string symbol_denom = 1;
+  uint32 exponent = 2;
+  string oracle_price = 3
+  [(gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec", (gogoproto.nullable) = true];
```

Comment:
Note how nullable is true here - if oracle is down, this field will be null but the query will work.